pages/src/navigation/sidebar/sidebar.ts

```diff
@@ -59,11 +60,13 @@ export class ESLDemoSidebar extends ESLToggleable {
   }
 
   protected onShow(params: SidebarActionParams) {
+    localStorage.setItem('sidebar-expanded', JSON.stringify(true));
```

Comment:
Can be simplified to storing single param 'sidebar-collapsed' that can be 'true' or undefined (removed)
There is no need an object or `JSON.parese`.
Moreover, we should have a sidebar opened by default for a first-time visits.